platform/darwin/src/MGLStyle.mm

```diff
@@ -92,6 +93,10 @@ - (NSString *)name {
     return @(self.mapView.mbglMap->getStyleName().c_str());
 }
 
+- (NSURL *)URL {
```

Comment:
Unexpected in what sense? The private definition of this property says it’s `nullable`. Is there an expectation that `URL` will ends up returning something in the future that isn’t a URL or `null`?